Directory.Build.targets

```diff
@@ -1,22 +1,28 @@
 <Project>
   <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
-    <MsbVer>17.8.3</MsbVer>
+    <MsbVer>17.8.43</MsbVer>
     <CscVer>4.8.0-3.final</CscVer>
     <ExtsVer>8.0.0</ExtsVer>
     <LncdCaVer>8.1.2496</LncdCaVer>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)' == 'net9.0'">
-    <MsbVer>17.12.6</MsbVer>
+    <MsbVer>17.12.50</MsbVer>
```

Comment:
Don't bump this, no reason to remove support for older SDKs. Same above.

Reply:
Does bumping patch version affect the SDK version (substantially)?